draft-ietf-secevent-delivery.xml

```diff
@@ -125,55 +125,21 @@
 					The following definitions are defined for Security Event distribution:
 					<list style="hanging">
           
-            <t hangText="Identity Provider"><vspace/>
-              An Identity Provider is a service provider that issues authentication
-              assertions that may be used by Relying Party service providers
-              to establish login sessions with users. Examples of Identity 
-              Providers are defined in: OpenID Connect 
-              <xref target="openid-connect-core"/> and SAML2 
-              <xref target="saml-core-2.0"/>. For the purpose of this 
-              specification an Identity Provider also includes any provider
-              of services where the compromise of an account may open up 
-              relying parties to attack. For example for the purposes of
-              security events, an email service provider could be 
-              considered an "implicit" Identity Provider.
-            </t>
-              
-            <t hangText="Relying Party"><vspace/>
-              Relying Parties come in multiple forms generally classified 
-              as "Explicit" or "Implicit". An Explicit Relying Party is a 
-              service provider that accepts a standard security 
-              assertion (e.g. a JWT access tokens <xref target="RFC7519"/>)
-              from an Identity Provider to establish a session or 
-              authorization. An Implicit Relying Party (implicit) uses 
-              a personal identifier such as an email address or telephone 
-              number from another provider to establish a Subject's 
-              identity. Examples of Explicit Relying Parties are defined 
-              in: OpenID Connect <xref target="openid-connect-core"/> and 
-              SAML2 <xref target="saml-core-2.0"/>. Implicit relying 
-              parties are verified by a common channel associated with 
-              the identifier. For example, an email or a text message is
-              sent with a unique link to establish ownership of the
-              identifier by the Subject.
-            </t>          
-            
             <t hangText="Event Transmitter"><vspace/>
               A service provider that delivers SETs to other providers known
-              as Event Receivers. Some examples of Event Transmitters are 
-              Identity Providers and Relying Parties. An Event Transmitter
+              as Event Receivers. An Event Transmitter
               is responsible for offering a service that allows the Event
               Receiver to check the Event Stream configuration and status
-              known as the "Control Plane". 
+              known as the "Management API". 
```

Comment:
Unfortunately the IETF does not like the term API. API is applied to a programming language or a library and not for a protocol.